eagle-hadoop-metric/src/main/webapp/app/apps/hadoop_metric/partials/overview.html

```diff
@@ -33,17 +59,19 @@ <h3 class="box-title">
 		<span class="fa fa-line-chart"></span>
 		<h3 class="box-title">
 			Metrics
+			<span>({{defaultHostname}})</span>
 		</h3>
 	</div>
 	<div class="box-body no-padding">
 		<div class="row border-split">
-			<div class="col-sm-6 col-md-4 col-lg-3" ng-repeat="metric in metricList track by $index">
+			<div class="col-sm-6 col-md-6 col-lg-3" ng-repeat="metric in metricList track by $index">
```

Comment:
May be `col-lg-4` instead? `col-lg-3` will much shorter when screen width from `col-md-6` to `col-lg-3`